pkg/process/checks/process.go

```diff
@@ -46,7 +46,7 @@ func NewProcessCheck(config ddconfig.ConfigReader) *ProcessCheck {
 	}
 
 	if workloadmeta.Enabled(config) {
-		check.workloadMetaExtractor = workloadmeta.NewWorkloadMetaExtractor(config)
+		check.workloadMetaExtractor = workloadmeta.NewWorkloadMetaExtractor(ddconfig.SystemProbe)
```

Comment:
Unfortunately, I'd have to modify a ton of code to inject the system probe in the constructor. Removing the `checks.All()` function would solve this constraint.